### PR TITLE
gooddata writer v3: simplify sso login retrieval

### LINKED ISSUE #2713: gooddata writer v3 - simplify SSO Login retrieval
- nebude Enable/Disable, ale jen Access Project vid https://github.com/keboola/gooddata-provisioning/pull/31

### PR DESCRIPTION
**Pull request by kacurez**
Fixes #2713 
related to backend change https://github.com/keboola/gooddata-provisioning/pull/31

Proposed changes:
- removes enable/disable sso access buttons and provisioning support
- loads sso login directly after page load or after gd project has been created

backend uprava este neni nasadena, pre vyskusanie je treba upravit provisioning url v
https://9jrqlbiw96.execute-api.us-east-1.amazonaws.com/test
tu https://github.com/keboola/kbc-ui/blob/kacurez-update-wrgdv3-provisioning/src/scripts/modules/provisioning/ProvisioningApi.js#L16

### CODE REVIEW

File: src/scripts/modules/gooddata-writer-v3/react/components/CredentialsContainer.jsx [hunk 1]
```diff
@@ -47,19 +46,14 @@ export default React.createClass({
...
-    });
-  },
-
   handleCreate(newProject) {
     if (newProject.isCreateNewProject) {
       const {name, tokenType, customToken} = newProject;
       return ProvisioningActions.createProject(name, tokenType, customToken).then(
-        ({pid, login, password}) => this.props.onSave({pid, login, password})
+        ({pid, login, password}) => {
+          ProvisioningActions.loadProvisioningData(pid);
```
Comment (ujovlado) [inline src/scripts/modules/gooddata-writer-v3/react/components/CredentialsContainer.jsx]: @kacurez len mikro vec, nemalo by sa zavolat `ProvisioningActions.loadProvisioningData` az ked sa to podari ulozit?

Comment (kacurez) [reply src/scripts/modules/gooddata-writer-v3/react/components/CredentialsContainer.jsx]: tu si myslim ze to je cajk, ak sa nepodari ulozit pid+credentials do konfigu tak dodatocne nacitanie infa o projekte sa nezobrazi. UI by malo ostat v init stave tj ze nema ziadny projekt nasetupovany + hlaska. Tiez som chcel tym podporit "happy path" kde by vytvorenie projektu netrvalo tak dlho(ze sa chainuju 3 api cally), to ze sa nepodari ulozit `onSave` je imho edge case

**Comment by ujovlado** [comment]
Kukam, ze gdwr je uz public beta - necham ten merge na vas, ak to chcete nejako subezne.

**Comment by kacurez** [comment]
ok urobime to dufam zajtra, diky

**Pull request merged**